### PR TITLE
Hot fix/first quest enforce char limit

### PR DESCRIPTION
**Pull request by tikisailor**
Enforce character limit and update the message content used during db initialization

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.2.1-RELEASE (2021-11-30)
```
Comment (amaredeus) [inline CHANGELOG.md:3]: Should be (2021-12-09)

**Review by amaredeus — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tikisailor) [reply CHANGELOG.md]: 2021-12-10 here ;)

Comment (tikisailor) [reply CHANGELOG.md]: 2021-12-10 here ;)

**Review by amaredeus — APPROVED** [review]

**Review by nonsensecodes — APPROVED** [review]
Looks good! Thank you @tikisailor!

**Pull request merged**